packages/stable/src/types.ts

```diff
@@ -2976,13 +2981,23 @@ export type TemplateGroup = ExternalTemplateGroup & {
    * When resource was last updated
    */
   lastUpdatedTime: Timestamp;
+
+  /**
+   * The dataSetId of the template group
+   */
+  dataSetId?: number;
```

Comment:
Does it not get tihs from the ExternalTemplateGroup?

Reply:
Ah yes, it does, thank you!